client/src/main/main.ts

```diff
@@ -3,10 +3,18 @@ import { EventEmitter } from '@genesislcap/foundation-events';
 import { App } from '@genesislcap/foundation-shell/app';
 import { importPBCAssets } from '@genesislcap/foundation-shell/pbc';
 import { configureDesignSystem } from '@genesislcap/foundation-ui';
-import { baseLayerLuminance, StandardLuminance } from '@microsoft/fast-components';
-import { FASTElement, customElement, observable, DOM } from '@microsoft/fast-element';
-import { Container, inject, Registration } from '@microsoft/fast-foundation';
-import { DefaultRouteRecognizer } from '@microsoft/fast-router';
+import {
+  baseLayerLuminance,
+  customElement,
+  Container,
+  DefaultRouteRecognizer,
+  DOM,
+  GenesisElement,
+  inject,
+  observable,
+  Registration,
+  StandardLuminance
+} from '@genesislcap/web-core';
```

Comment:
Fyi, we run eslint/prettier on the generated app during CI, so any changes like this will need to pass

Reply:
Okay cool...